cdec/installation-manager-core/src/main/java/com/codenvy/im/artifacts/helper/CDECSingleServerHelper.java

```diff
@@ -535,6 +539,12 @@ public Command getReinstallCommand(Config config, @Nullable Version installedVer
     public Command getUpdateHostnameCommand(Config config, String oldHostName, String newHostName) {
         List<Command> commands = new ArrayList<>();
 
+        // add <127.0.0.1 new-hostname> record into the /etc/hosts on master
+        commands.add(createAppendTextToFileCommand(Paths.get("/etc/hosts"),
```

Comment:
I will change name of method *'createAppendTextToFileCommand()'* on *'createAppendTextIfAbsentToFileCommand()'*.